packages/headless/src/modules/cart/graphql/cart.graphql

```diff
@@ -0,0 +1,40 @@
+type Product {
+  name: String!
+  price: Int!
+  id: String!
+}
+
+type Item {
+  productId: String!
+  product: Product!
+  quantity: Int!
+}
+
+input ItemInput {
+  productId: String!
```

Comment:
remove ! here.because we have added JoiValidation schema in the service layer for input validation.